camera_system/camera_system.py

```diff
@@ -19,22 +21,24 @@ def __init__(self, is_left_course: bool = True) -> None:
         """
         self.__set_is_left_course(is_left_course)
 
-    def start(self) -> None:
+    def start(self, camera_id=1) -> None:
         """ゲーム攻略を計画する."""
-        # ToDo: キャリブレーションする(ゲームエリア情報).
-
+        # カメラキャリブレーションを開始する
+        camera_calibrator = CameraCalibrator(camera_id)
+        # GUIから座標取得
+        camera_calibrator.start_camera_calibration()
         # 通信を開始する.
         client = Client("127.0.0.1", 8080)
         # 開始合図を受け取るまで待機する.
         client.wait_for_start_signal()
-
-        # ToDo: ゲームエリア情報を生成する(ゲームエリア情報).
-
+        # ゲームエリア情報の作成
+        camera_calibrator.make_game_area_info()
         # ToDo: 計画する.
         game_motion_list = CompositeGameMotion()  # TODO: 計画した結果のゲーム動作のリストをセットする
 
         # コマンドファイルを生成する
-        file_name = "GameAreaLeft.csv" if self.is_left_course else "GameAreaRight.csv"  # ファイル名をセット
+        os.mkdir("command_files")
```

Comment:
これなら大丈夫かな？
[参考](https://note.nkmk.me/python-os-mkdir-makedirs/)
```suggestion
        os.makedirs("command_files", exist_ok=True)
```